Evan-Sara-Jalal/movie-app/src/components/movieGrid.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Movie from "./Movie";
+const MovieGrid = props => {
+  return (
+    <div className="container">
+      <div className="row mt-5">
+        {props.movies.map((movie, index) => {
+          return (
+            <Movie
+              image={movie.poster_path}
+              title={movie.title}
+              overview={movie.overview}
+              popularity={movie.popularity}
+              release_date={movie.release_date}
+              key={index}
```

Comment:
for a short cut, you can pass the movie object to Movie component and in that component put what you want to you form it 
ex: 
` {props.movies.map((movie) => (<Movie movie={movie} />))}
`

in Movie component : 

```
<h1>{movie.original_title}</h1>
<p>{movie.release_date}</p>
```
and so on ...